study.md

```diff
@@ -43,7 +43,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+`lenth` just gives the number of elements in an array, but even if one of those elements is another array it will count it as 1 instead of as the number of things inside the array element.
```

Comment:
This does not answer the question.

`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.